installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -0,0 +1,116 @@
+[id="installing-gcp-user-infra-vpc"]
+= Installing a cluster on GCP using Deployment Manager templates
+include::modules/common-attributes.adoc[]
+:context: installing-gcp-user-infra-vpc
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on
+Google Cloud Platform (GCP) that uses infrastructure that you provide.
+
+The steps for performing a user-provided infrastructure install are outlined here. Several
+link:https://cloud.google.com/deployment-manager/docs[Deployment Manager] templates are provided to assist in
+completing these steps or to help model your own. You are also free to create
+the required resources through other methods; the templates are just an
+example.
+
+.Prerequisites
+
+* Review details about the
+xref:../../architecture/architecture-installation.adoc#architecture-installation[{product-title} installation and update]
+processes.
+* You must have an existing VPC in GCP that contains separate subnets for the compute and control plane machines that the cluster requires.
```

Comment:
Although the shared vpc alongside host project should be pre-existing resources, IMO, we still need to provide an end-to-end solution to customers so that they can deploy OCP from scratch. Therefore, I suggest to remove this line and add a section to describe how to configure host project and shared VPC.

Reply:
@shellyyang1989 or @jstuever, do you have more details about the host project and VPC configuration?

Reply:
To learn more about how to enable shared VPC, please refer to https://cloud.google.com/vpc/docs/provisioning-shared-vpc#setting_up. Once shared VPC is enabled, all the vpc networks in host project can be accessed by service project.
The host project is dedicated to host vpc network and dns which do not have much difference with those in stand-alone project.
The host project should have these:
1. Service account which has these roles enabled:
    Compute Network User
    Compute Security Admin
    Deployment Manager Editor
    DNS Administrator
    Security Admin
    Network Management Admin 
2. Have user managed service account of service project got 'Compute Network User' role of host project
3. Have google managed service account i.e. project-number@cloudservices.gserviceaccount.com of service project got 'Compute Network User' role of host project
4. A public dns zone for external dns resolution
5. VPC network